cfgov/jobmanager/tests/test_models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import IntegrityError
+from django.test import TestCase
+from model_mommy import mommy
+
+from jobmanager.models import Job
+
+
+class JobTestCase(TestCase):
+    def test_slug_uniqueness(self):
+        mommy.make(Job, slug='abcde')
+        with self.assertRaises(IntegrityError):
+            mommy.make(Job, slug='abcde')
+
+    def test_slug_case_sensitive_uniqueness(self):
+        mommy.make(Job, slug='abcde')
+        mommy.make(Job, slug='ABCDE')
```

Comment:
It's verifying that you can create two `Job` objects with slugs that differ only by case. The test only passes if these two don't violate the uniqueness constraint (which means that differing cases are unique). Without that (default MySQL behavior, without this new migration) the second `make` call will raise an `IntegrityError`.